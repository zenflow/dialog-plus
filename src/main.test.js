/* eslint-env jest */

import DialogPlus from './main.js'

describe('DialogPlus', () => {
  it('DialogPlus(string)', () => {
    expect(DialogPlus('cool')).toMatchSnapshot()
  })
})
