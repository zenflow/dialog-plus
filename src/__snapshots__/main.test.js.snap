// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DialogPlus DialogPlus(string) 1`] = `
Object {
  "doubled": "coolcool",
  "string": "cool",
}
`;
